rsign/signature.py

```diff
@@ -70,7 +72,10 @@ def compare(self, s1, s2):
     def sign_string(self, key, text):
```

Comment:
What type does sign_string return?  Add to the comment and for corresponding sign_request method in request.py.

Reply:
Would you prefer if we returned either string or bytes depending on your python version, or just force the result to always be string?

Returning bytes in py3 would be the correct result imo, but it would complicate using the library.